### ISSUE TITLE
Bug in run_cmds -p when path is invalid

### ISSUE THREAD

**Issue by jathanism** [trigger/trigger#217]
```
$ run_cmds -p invalid
/usr/lib/python2.7/dist-packages/zope/__init__.py:3: UserWarning: Module twisted was already imported from /usr/lib/python2.7/dist-packages/twisted/__init__.pyc, but /usr/local/lib/python2.7/dist-packages is being added to sys.path
  import pkg_resources
Traceback (most recent call last):
  File "/usr/local/bin/run_cmds", line 10, in <module>
    docommand.main(action_class=docommand.CommandRunner)
  File "/usr/local/lib/python2.7/dist-packages/trigger/contrib/docommand/core.py", line 88, in main
    opts, args = parse_args(sys.argv, description=action_class.description)
  File "/usr/local/lib/python2.7/dist-packages/trigger/contrib/docommand/core.py", line 402, in parse_args
    ok, err = verify_opts(opts)
  File "/usr/local/lib/python2.7/dist-packages/trigger/contrib/docommand/core.py", line 429, in verify_opts
    return False, 'ERROR: %r is not a valid path\n' % path
NameError: global name 'path' is not defined
```

**Comment by coxley** [comment]
Seems like a bug for any provided path, looks like an easy fix. Will submit PR momentarily.

**Comment by jathanism** [comment]
Fixed in #217
